README.md

```diff
@@ -19,3 +19,12 @@ https://welcome.realdevsquad.com/discord
 # Code of conduct at RDS
 
 https://welcome.realdevsquad.com/code-of-conduct.html
+
+# Running the project
+
+- Run `npm install` after opening the repository in your editor. This will install all the necessary packages for the project which are mentioned in the `package.json` file of the project.
+
```

Comment:
Can we please link `CONTRIBUTING.md` here? The first 3 steps are related to the step up itself
Once the devs have the setup done, we can ask them to do `npm install`

Reply:
Sure. It will be of great help! 👍